.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: Stale
+
+on:
+  issues:
+    types: [reopened]
+  schedule:
+    - cron: '*/60 * * * *'
```

Comment:
this runs every 60 minutes? Is this the default? Not sure if that is maybe a bit overkill?

Reply:
changed to `'30 1 * * *'`